.chronus/changes/vs-code-extension-typespec-snippets-2024-9-15-11-2-20.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: feature
+packages:
+  - typespec-vscode
+---
+
+feat(vscode): add basic snippets for typespec
```

Comment:
please don't name things like that this ends up in the changelog. THis should be descriptive markdown

Reply:
same for the pr title in general we don't use that git changelog system so its not really necessary

Reply:
updated changeset and pr title accordingly